packages/marketing-analytics-browser/src/browser-client.ts

```diff
@@ -1,52 +1,21 @@
 import { createInstance as createBaseInstance } from '@amplitude/analytics-browser';
 import { returnWrapper } from '@amplitude/analytics-core';
-import { webAttributionPlugin } from '@amplitude/plugin-web-attribution-browser';
 import { context } from './plugins/context';
-import { Client, Options } from './typings/browser-client';
-import { BrowserOptions } from '@amplitude/analytics-types';
+import { BrowserClient, BrowserOptions } from '@amplitude/analytics-types';
 
-export const createInstance = (): Client => {
+export const createInstance = (): BrowserClient => {
   const client = createBaseInstance();
 
-  const _init = async (options: Options & { apiKey: string }) => {
-    const { attribution, pageViewTracking, apiKey, userId, ...restOfOptions } = options;
+  const _init = async (options: BrowserOptions & { apiKey: string }) => {
```

Comment:
note: removes marketing analytics specific options and behavior

Reply:
@yuhao900914 , think we could update GTM templates too and help make this happen? Think that was the only dependency left? Reading Kevin's summary it sounds like 2.0 now matches MA's web attribution tracking behavior so might not even need a major version bump.

Reply:
The default behavior for web attribution is different. like in the browser the default behavior is `trackNewCampaigns = false`, it's true in marketing analytics SDK. Also, the default event name has changed too, if they didn't manually set the event_type for the page view event, their chart might be broken. 
But I just feel we can directly twist it in the GTM template to make those default behavior the same as previously.  
@kevinpagtakhan  Did I miss anything?